portfolio/render.go

```diff
@@ -29,28 +32,33 @@ func NaNString() string {
 	return "NaN"
 }
 
-func (h _PrintHelper) CurrStr(val decimal.Decimal) string {
+func (h PrintHelper) CurrStr(val decimal.Decimal) string {
+	v, err := strconv.ParseFloat(val.String(), 64)
+	if err != nil {
+		panic(err)
+	}
+	p := message.NewPrinter(language.English)
 	if h.PrintAllDecimals {
-		return val.String()
+		return p.Sprintf("%f", v)
 	}
```

Comment:
Why? Converting this back to float will break the decimal encoding.

Reply:
Reverted in the latest version.